### PR TITLE
Connection to a self-hosted lavalink

### PR DESCRIPTION
**Pull request by fpekal**
Changed config.js to automatically connect to a lavalink from a docker-compose image.

## Please describe the changes this PR makes and why it should be merged:

Previously you had to change config.js to use a built-in lavalink node.
With this change it will be easier to setup for *not-so-geeky* users.

## Status and versioning classification:

- There are no code changes
- Typings don't need updating
- This PR **only** includes non-code changes, like changes to documentation, README, etc.

## Summary by CodeRabbit

- **Chores**
  - Updated default settings for server connectivity to enhance user experience.

### CODE REVIEW

**Comment by Neko-Life** [comment]
@BioCla why this repo isn't archived yet 🧐

File: config.js [hunk 1]
```diff
@@ -26,9 +26,9 @@ module.exports = {
 	nodes: [
 		{
 			identifier: "Main Node", //- Used for indentifier in stats commands.
-			host: "", //- The host name or IP of the lavalink server.
-			port: 80, // The port that lavalink is listening to. This must be a number!
-			password: "", //- The password of the lavalink server.
+			host: "lavalink", //- The host name or IP of the lavalink server.
+			port: 2333, // The port that lavalink is listening to. This must be a number!
+			password: "bonsoirDocker", //- The password of the lavalink server.
```
Comment (Neko-Life) [inline config.js:31]: this file is meant to be configured by users and not everyone have the exact same setup as you so this change is useless, also this version of the bot is obsolete. Checkout https://github.com/wtfnotavailable/Discord-MusicBot for the latest version of the bot with complete dashboard

Comment (fpekal) [reply config.js:31]: I understand that they will probably have a different config, provided they are using non-docker version.
But when someone would use `docker-compose up` they will be using lavalink config from `docker/application.yml`.

Nothing will change for non-docker users; they still have to change lavalink config.
But for docker-compose users everything would be already running.

**Comment by pheeef** [comment]
Changing the Default Port is fine, although this should be solved with Variables. But as Neko-Life already said this Version of the bot is not under active development anymore.

**Pull request closed without merging**